CoverMyDotNet/Requests/SearchForm.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using RestSharp;
+
+namespace CoverMyDotNet.Requests
+{
+	public class SearchForm : RestRequest
```

Comment:
Is there a reason that we have one class per API method rather than one per API?  Perhaps there is a technical reason imposed by RestSharp, or was this the desired usage?